docs/validators/changeset.rst

```diff
@@ -25,6 +25,10 @@ Change set
       max:
          count: 2 # max number of files in a PR
          message: 'Custom message...'
+      files:
```

Comment:
It would be helpful if you can add an comment about what this setting is for. just for clarity sake.